lib/author.rb

```diff
@@ -0,0 +1,15 @@
+class Author
+  attr_accessor :first_name, :last_name
+  attr_reader :id, :items
+
+  def initialize(first_name, last_name)
+    @first_name = first_name
+    @last_name = last_name
+    @id = rand(1..1000)
+    @items = []
+  end
+
+  def add_item(item)
+    @items << item unless @items.include?(item)
+  end
```

Comment:
Good job. Please consider the other side of the relationship: the Item. If we add an Item to an Author, the Author should be added to the associated Item as well.